lib/adyen/services/checkout.rb

```diff
@@ -12,9 +12,7 @@ def initialize(client, version = DEFAULT_VERSION)
         :sessions
       ]
 
-      with_application_info = [
-        :payment_session,
```

Comment:
why remove this?

Reply:
Can't add application info to the requests anymore as that's removed in checkout v70.